Xamarin.Forms.Core/Button.cs

```diff
@@ -245,66 +234,20 @@ protected override void OnBindingContextChanged()
 		void IFontElement.OnFontChanged(Font oldValue, Font newValue) =>
 			InvalidateMeasureInternal(InvalidationTrigger.MeasureChanged);
 
-		Aspect IImageController.Aspect => Aspect.AspectFit;
-		ImageSource IImageController.Source => Image;
-		bool IImageController.IsOpaque => false;
-
-		BindableProperty IImageController.SourceProperty => ImageProperty;
-		BindableProperty IImageController.AspectProperty => null;
-		BindableProperty IImageController.IsOpaqueProperty => null;
-
-
-		void IImageController.RaiseImageSourcePropertyChanged() => OnPropertyChanged(ImageProperty.PropertyName);
+		Aspect IImageElement.Aspect => Aspect.AspectFit;
+		ImageSource IImageElement.Source => Image;
+		bool IImageElement.IsOpaque => false;
```

Comment:
so the IImageElement is not fully implemented with BindableProperties here ?

Reply:
No, these are just the defaults that all these were already set to on Button when working with the Image.  They could become bindable properties but that's probably for a different PR